src/dashboard/src/templates/ingest/as/match.html

```diff
@@ -16,7 +16,12 @@
   function toggle_objects(source) {
     checkboxes = document.querySelectorAll('.atk-matcher-object-path > input');
     for (var i = 0, n = checkboxes.length; i < n; i++) {
-      checkboxes[i].checked = source.checked;
+      // Only select if the checkbox is visible and not disabled
+      if ( $(checkboxes[i]).is(":visible") && ! $(checkboxes[i]).is(":disabled") ) {
```

Comment:
Instantiating jQuery objects is slow (I think that's the reason why `this` in `jQuery.each` is a DOM object and not a jQuery object). Given that this is happening inside a loop it may be worth to optimize a bit by storing a reference in a variable and use it from your two conditions of the `if`.